From 607b1ff5992cac808f588abe570983a0132928a2 Mon Sep 17 00:00:00 2001
From: Anton Golubev <golubevan@altlinux.org>
Date: Wed, 3 Jul 2024 10:42:24 +0300
Subject: [PATCH] hwdb: Add some HP IR cameras

Two very similar devices, with two functions - a regular camera and IR.
The peculiarity of their infrared camera is that it uses a color image
format (YUYV), although it is essentially black and white.

The IR camera interface differs from the regular camera interface by name:

"HP Wide Vision FHD Camera: HP W" for the regular camera and
"HP Wide Vision FHD Camera: HP I" for an infrared camera

Therefore, glob *I is used to separate the IR camera

(cherry picked from commit 4ee0ac1ae426f4169a6c79ac732a6da903ca218f)

Related: RHEL-5950
---
 hwdb.d/70-cameras.hwdb | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/hwdb.d/70-cameras.hwdb b/hwdb.d/70-cameras.hwdb
index 3a84792a18..0fd0baffc8 100644
--- a/hwdb.d/70-cameras.hwdb
+++ b/hwdb.d/70-cameras.hwdb
@@ -12,6 +12,16 @@
 camera:usb:v*p*:name:*IR Camera*:
  ID_INFRARED_CAMERA=1
 
+###########################################################
+# Hewlett-Packard
+###########################################################
+
+# Chicony Electronics Co., Ltd HP Wide Vision FHD Camera (IR function)
+camera:usb:v04f2pb634:name:*I:
+# Realtek Semiconductor Corp. HP Wide Vision FHD Camera (IR function)
+camera:usb:v0bdap58e6:name:*I:
+ ID_INFRARED_CAMERA=1
+
 ###########################################################
 # Philips
 ###########################################################
